packages/client-account/src/methods/accountCopyIndex.ts

```diff
@@ -58,7 +58,20 @@ export const accountCopyIndex = (
     )
```

Comment:
the steps above here don't all need to happen in separate promise steps, but could be done in a Promise.all (unless you have a reason to want this to happen sequentially

Reply:
this to happen sequentially

Reply:
They need to happen sequentially.